ngx-fudis/projects/ngx-fudis/src/lib/components/divider/divider.docs.mdx

```diff
@@ -0,0 +1,26 @@
+import { ArgTypes, Meta, Canvas } from "@storybook/blocks";
+import { DividerComponent } from "./divider.component";
+import * as DividerStories from "./divider.stories.ts";
+
+<Meta title="Components/Divider" />
+
+# Divider
+
+Fudis Divider is a horizontal rule that implies a thematic break between HTML paragraph-level elements.
```

Comment:
We don't need to repeat Fudis prefix in our documentation. I would change Fudis Divider --> Divider